### PR TITLE
Changed virtualbox download URL to https.

### PR DESCRIPTION
**Pull request by mikecole**

### CODE REVIEW

File: ketarin/virtualbox.ketarin.xml [hunk 1]
```diff
@@ -79,7 +79,7 @@
     <FileHippoId />
     <LastUpdated>2013-07-05T06:10:01.4628832</LastUpdated>
     <TargetPath>C:\Chocolatey\_work\</TargetPath>
-    <FixedDownloadUrl>http://download.virtualbox.org/virtualbox/{version3Levels}/VirtualBox-{realVersion}-Win.exe</FixedDownloadUrl>
+    <FixedDownloadUrl>https://download.virtualbox.org/virtualbox/{version3Levels}/VirtualBox-{realVersion}-Win.exe</FixedDownloadUrl>
```
Comment (ferventcoder) [inline ketarin/virtualbox.ketarin.xml]: Did they fix this recently? It was mentioned and I thought I saw that while https://virtualbox.org works, https://download.virtualbox.org does not.

**Comment by mikecole** [comment]
You are correct. Sorry. :grimacing:

**Comment by ferventcoder** [comment]
No worries

**Comment by ferventcoder** [comment]
I had originally made the same change as well and then someone mentioned to me that it wouldn't work. :/

**Pull request closed without merging**